src/gui/tray/UserModel.cpp

```diff
@@ -667,6 +690,23 @@ Q_INVOKABLE bool UserModel::isUserConnected(const int &id)
     return _users[id]->isConnected();
 }
 
+Q_INVOKABLE bool UserModel::isNotificationStatusOnline(const int &id)
```

Comment:
Any reason why it's const int &id instead of const int id? I think a int is even smaller than the address on 64-bit systems.